poutyne/framework/model.py

```diff
@@ -219,9 +226,11 @@ def fit_generator(self, train_generator, valid_generator=None, *,
 
         Args:
             train_generator: Generator-like object for the training dataset.
-                The generator must yield a tuple ``(x, y)`` where ``x`` is a
+                The generator must yield a tuple ``(x_1, x_2, ..., x_n, y)`` where ``x`` is a
```

Comment:
Write something else than ``x`` since ``x`` is not defined here anymore.

Reply:
How about this?

The generator must yield a tuple ``(x_1, x_2, ..., x_n, y)`` where ``x_i`` are ``n`` inputs to the model and ``y`` is the corresponding ground truth.  Each ``x_i`` and ``y`` should be a Tensor or a Numpy array with the first dimension being the batch size since ``len(y)`` is taken as the batch size.

Reply:
Just change the first sentence to this:

> The generator must yield a batch in the form of a tuple ``(x_1, x_2, ..., x_n, y)`` where the ``x_i``s are ``n`` inputs and ``y`` is the corresponding ground truths.

The thing is I don't want the users constrained to put tensors into the tuple since it could contain anything like objects or dictionaries. Anyway, the phrasing was already not good to start with. I will change it in my free time.